Always_green_CI.md

```diff
@@ -0,0 +1,56 @@
+# Introduction
+
+We have occasionally seen that the CI fails due to flaky tests or jobs, environment issues and more. This presents a challenge for new contributors as they need to spend time figuring out issues that are not related to their changes in the code. It also hampers the confidence the community has in the CI as a simple yes/no answer and instead talks about must vs optional tests. We believe that this will become an increasing bottleneck if the kata community is successful in kicking off the new confidential containers project which will bring in additional participating companies and individual contributors. Given this, we established a process and policy that should allow the project to maintain a "Always Green CI" aiming to improve the results of Kata Containers CI shown in GitHub.
+
+# Concepts
+
+In this section is discussed the main pillars that sustain the "Always Green CI" initiative.
+
+## Baseline jobs
+
+There was created multiple jobs (called "baselines") which run on daily basis for the different test suites and configuration/HW which are identical to what will run on each pull request (PR). The key point is that this CI runs on a branch with no additional patches/noise added by anyone so if there is a failure it’s much easier to pinpoint it. The baseline jobs indicate the overall status of the Kata Containers project and we pursue to keep them "always green". They are used as a base for comparing with the CI jobs ran on each pull request (PR): if a given CI job fails but its corresponding on baseline is "green" then it is likely the code change breaking tests. 
```

Comment:
Change "There was created" to "We have created"

Reply:
- s/run on daily/run on a daily/
- s/pinpoint/identify/
- s/we pursue to keep them "always green"/our aim is to ensure they always pass ("always green")/ maybe?

Reply:
Done